src/ui/server/helpers/object/index.ts

```diff
@@ -9,6 +9,10 @@ type ObjectType = {
  * @returns {Boolean}
  */
 export const objectHasKeysAndValues = (obj: ObjectType) => {
+  if (!obj) {
+    return false;
+  }
```

Comment:
I don't think this is needed, `obj` is a required param so if it's not provided it won't compile

Reply:
I can remove this.  I had it crash on me once when for some reason when I passed the wrong variable and it got undefined and went to problem with service.  What do you say?

Reply:
I think it's safe to remove, I don't believe it's possible for it to error if object is not passed - if the function is called without an object it will not compile. Unless you can replicate it erroring seems ok to remove